README.md

```diff
@@ -35,17 +35,21 @@ NB : System apps cannot really be uninstalled without root (see the [FAQ](https:
 * [ ] Gionee
 * [X] LG
 * [X] Google
+* [ ] iQOO
 * [X] Fairphone
 * [ ] HTC
 * [X] Huawei
 * [X] Motorola
 * [X] Nokia
```

Comment:
```suggestion
* [X] Nokia
* [X] Nothing
```

See https://github.com/0x192/universal-android-debloater/issues/381 and https://github.com/0x192/universal-android-debloater/issues/381#issuecomment-1243579483.